src/Config.php

```diff
@@ -58,6 +58,7 @@ public function getRules() : array {
 			'single_import_per_statement' => true,
 			'single_line_after_imports' => true,
 			'switch_case_space' => true,
+			'types_spaces' => true,
```

Comment:
Allowed value in docs is `'none'`

Should also do it for multiple catches

Reply:
true = enable rule with defaults (space = none, space_multiple_catch = follow space)

I tested it locally, and it changes the code like: 

![image](https://github.com/nextcloud/coding-standard/assets/3902676/6cb4cf04-bad6-4dba-be37-7121d2d40e72)
![image](https://github.com/nextcloud/coding-standard/assets/3902676/5c51efcd-57c0-49ab-bde7-530af29a951c)

If you prefer, I can update the configuration to the more explicit array notation.